src/appmixer/pipedrive/crm/FindPerson/component.json

```diff
@@ -4,37 +4,73 @@
     "description": "This action finds person.",
     "inPorts": [
         {
-            "name": "query",
+            "name": "in",
             "schema": {
                 "type": "object",
                 "properties": {
                     "term": {
                         "type": "string",
                         "minLength": 2
                     },
-                    "orgId": { "type": "number" },
-                    "searchEmail": { "type": "boolean" }
+                    "orgId": {
+                        "type": "number"
+                    },
+                    "exactMatch": {
+                        "type": "boolean"
+                    },
+                    "outputType": {
+                        "type": "string"
+                    }
                 },
-                "required": ["term"]
+                "required": [
+                    "term"
+                ]
             },
             "inspector": {
                 "inputs": {
+                    "outputType": {
+                        "type": "select",
+                        "label": "Output Type",
+                        "index": 0,
```

Comment:
`outputTypes` should be below `term`, the search term is more important.

Reply:
True but my idea was that it is more important to first setup how you want the component to return the values and then have the usable features of the component. Basically have the select more visible and make sure nobody misses that you can change how the component output will show.